frontend/src/app/search/actions.ts

```diff
@@ -1,15 +1,10 @@
 // All exports in this file are server actions
 "use server";
 
-import { APISearchFetcher } from "../../services/searchfetcher/APISearchFetcher";
-import { MockSearchFetcher } from "../../services/searchfetcher/MockSearchFetcher";
-import { fetchSearchOpportunities } from "../../services/searchfetcher/SearchFetcher";
+import { getSearchFetcher } from "../../services/searchfetcher/SearchFetcherUtil";
 
-const useMockData = false;
-const searchFetcher = useMockData
-  ? new MockSearchFetcher()
-  : new APISearchFetcher();
+const searchFetcher = getSearchFetcher();
 
 export async function updateResults() {
-  return await fetchSearchOpportunities(searchFetcher);
+  return await searchFetcher.fetchOpportunities();
```

Comment:
PReviously we had a helper function in `SearchFetcher.ts` that called `fetchOpportunities` - but it was just an intermediary that is unnecessary